packages/shared/components/wallet-widget/wallet-widget.tsx

```diff
@@ -86,7 +86,7 @@ class _WalletWidget extends React.PureComponent<
               </StatisticItem>
             </div>
             <div className="wallet-details__item">
-              <StatisticItem label={t("wallet-widget.invested-value")}>
+              <StatisticItem label={t("wallet-widget.invested")}>
```

Comment:
It breaks the path to translation.
![image](https://user-images.githubusercontent.com/27726590/58862813-31db3780-86ba-11e9-8d31-311d6e572d64.png)